### PR TITLE
Made the README look nicer

### PR DESCRIPTION
**Pull request by sdushantha**
# Description

- Centered the logo and badges
- Added table of content
- Added _back to top_ link

## Status
**READY**

## Type of change
- Documentation (fix or adds documentation)

# How Has This Been Tested?
I opened the README file and looked at the pretty documentation :)

# Checklist:

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation (if applicable)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,94 +1,122 @@
...
-
-This application is designed to take messages from chat services and execute python functions (skills) based on their contents. Those functions can be anything you like, from simple conversational responses to running complex tasks. The true power of this project is to act as a glue library to bring the multitude of natural language APIs, chat services and third party APIs together.
+<h6 align=center>
+<img src="https://github.com/opsdroid/style-guidelines/raw/master/logos/logo-wide-light.png"/>
+</h6>
+
+<h4 align=center>An open source chat-ops bot framework</h4>
+
+<p align=center>
+<a href="https://pypi.python.org/pypi"><img src="https://img.shields.io/pypi/v/opsdroid.svg" alt="Current version of pypi" /></a>
```
Comment (jacobtomlinson) [inline README.md]: Why are these badges now in HTML and now markdown?

File: README.md [hunk 2]
```diff
@@ -1,94 +1,122 @@
...
-[![Gitter Badge](https://img.shields.io/badge/gitter-join%20chat-4fb896.svg)](https://gitter.im/opsdroid)
-[![Backers on Open Collective](https://opencollective.com/opsdroid/backers/badge.svg)](#backers)
-[![Sponsors on Open Collective](https://opencollective.com/opsdroid/sponsors/badge.svg)](#sponsors)
-[![Open Source Helpers](https://www.codetriage.com/opsdroid/opsdroid/badges/users.svg)](https://www.codetriage.com/opsdroid/opsdroid)
-
-An open source chat bot framework written in python. It is designed to be extendable, scalable and simple.
-
-This application is designed to take messages from chat services and execute python functions (skills) based on their contents. Those functions can be anything you like, from simple conversational responses to running complex tasks. The true power of this project is to act as a glue library to bring the multitude of natural language APIs, chat services and third party APIs together.
+<h6 align=center>
+<img src="https://github.com/opsdroid/style-guidelines/raw/master/logos/logo-wide-light.png"/>
```
Comment (jacobtomlinson) [inline README.md]: Could you add an alt tag?

File: README.md [hunk 3]
```diff
@@ -1,94 +1,122 @@
...
+
+An open source chat bot framework written in Python. It is designed to be extendable, scalable and simple.
+
+
+This application is designed to take messages from chat services and execute Python functions (skills) based on their contents. Those functions can be anything you like, from simple conversational responses to running complex tasks. The true power of this project is to act as a glue library to bring the multitude of natural language APIs, chat services and third party APIs together.
 
 Help support opsdroid in one click by pressing [![Tweet](https://img.shields.io/twitter/url/http/shields.io.svg?style=social)](https://twitter.com/intent/tweet?text=Check%20out%20opsdroid,%20an%20awesome%20open%20source%20chatbot%20framework%20written%20in%20Python.&url=https://opsdroid.github.io/&via=opsdroid&hashtags=chatbots,chatops,devops,automation,opensource)
 
 ## ChatOps
+[(Back to top)](#)
```
Comment (jacobtomlinson) [inline README.md]: As the README isn't very long I'm not sure these links are helpful, they are just adding visual clutter.

Could you please remove them?

File: README.md (hunk 1 above)
Comment (sdushantha) [reply README.md]: So that I can center them. Markdown does not allow me to do that.

File: README.md (hunk 2 above)
Comment (sdushantha) [reply README.md]: Done! 😄

File: README.md (hunk 3 above)
Comment (sdushantha) [reply README.md]: Done :)

**Comment by sdushantha** [comment]
@jacobtomlinson Oops.... looks like I missed that one. But I fixed it now :)

File: README.md [hunk 4]
```diff
@@ -134,7 +157,7 @@ Do you need help? Do you want to chat? [Join our Gitter channel](https://gitter.
 
 _\* databases are optional, however bot memory will not persist between different connectors or system reboots without one_
 
-## Contributors
+### Contributors
```
Comment (jacobtomlinson) [inline README.md]: Why have you chosen to indent this further? This makes it appear to be a subtitle of "Usage", which it isn't.

File: README.md [hunk 5]
```diff
@@ -53,39 +76,39 @@ Check out the [Getting Started](https://www.youtube.com/watch?v=7wyIi_cpodE&list
...
 ### Ubuntu 16.04 LTS
 
 ```bash
 # Update apt-get
-sudo apt-get update
+$ sudo apt-get update
 
-# Install pip for python3 and locales
-sudo apt-get install python3-pip language-pack-en git
+# Install pip for Python3 and locales
```
Comment (jacobtomlinson) [inline README.md]: Originally this was `python3` as in the binary. If we are capitalizing we should also add a space and make it `Python 3` as in the proper name.

File: README.md (hunk 4 above)
Comment (jacobtomlinson) [reply README.md]: Ignore this, GitHub had collapsed things and I got confused 🙃

File: README.md (hunk 5 above)
Comment (sdushantha) [reply README.md]: Agreed 👍

**Comment by sdushantha** [comment]
@jacobtomlinson Ready to merge? 😄

**Comment by jacobtomlinson** [comment]
Ping me your address via DM on [Twitter](https://twitter.com/_jacobtomlinson) or [Gitter](https://gitter.im/opsdroid) and I'll post you some [opsdroid stickers](https://medium.com/opsdroid/contributor-sticker-packs-738058ceda59)!

**Pull request merged**